pkgs/development/python-modules/nimbo/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, python3Packages, fetchFromGitHub, awscli }:
+
+with python3Packages;
+
+buildPythonApplication rec {
+  pname = "nimbo";
+  version = "0.2.2";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "nimbo-sh";
+    repo = "nimbo";
```

Opinionated change...

```suggestion
    repo = pname;
```